### PR TITLE
My Portfolio Project: Contact form

### PR DESCRIPTION
**Pull request by alexpuente0**
in this pull request, we:

- created a contact me section and form
- styled the contact form
- loaded new assets for the new section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,5 +243,94 @@ <h2>Skills</h2>
...
+        action="https://formspree.io/f/mlezlyew"
+        method="POST"
+      >
+        <input
+          class="contact-name celement"
+          type="text"
+          required
+          maxlength="30"
+          placeholder="Full name"
+        />
```
Comment (medaminedev66) [inline index.html]: - [x] The form submission doesn't work as expected. I would recommend adding `name="name"` to the input of name to be able to submit the data to the Formspree.

image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @alexpuente0 and @kinginthenorthcodez 😃, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️You have a descriptive PR

## Required Changes ♻️


_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (alexpuente0) [reply index.html]: just noticed this a minute after I hit submit for review, but I corrected it now, thanks for the feedback

**Review by rotimiazeez — APPROVED** [review]
# Hi Team, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**